nsot/models.py

```diff
@@ -1396,8 +1396,43 @@ def set_addresses(self, addresses, overwrite=False, partial=False):
 
         self.clean_addresses()
 
+    def get_ancestors(self):
+        """Recursively get all parents of an interface"""
+        p = self.parent
+        ancestors = []
```

Comment:
Hi @jathanism, how about I do the following. I get all the id's of the ancestors in a list like the following 

`ancestor_ids = [a.id for a in ancestors]`

Then I return the `Queryset` from the following query

`Interface.objects.filter(id__in=ancestor_ids)`